resources/AdvancedTabComponents/NtripClientTab.qml

```diff
@@ -13,15 +13,94 @@ Item {
     property var floatValidator
     property var intValidator
     property var stringValidator
+    property string url
+    property string mountpoint
+    property string port
 
     RowLayout {
         anchors.fill: parent
 
         ColumnLayout {
+            RowLayout {
+                ComboBox {
+                    Layout.fillWidth: true
+                    editable: true
+                    validator: stringValidator
+                    onAccepted: url = currentText
+
+                    model: ListModel {
+                        ListElement {
+                            text: "na.l1l2.skylark.swiftnav.com"
```

Comment:
This looks great, but I don't think it's what we need for this feature, we want to allow users to connect other NTRIP providers, and we (internally) want to be able to connect to internal staging or preproduction environments.

If we had a way to only conditionally show this combo box UI then it could be great.  For instance of someone selected "Skylark" as the ntrip type vs "Other".

For now I would just leave this as an experiment.